frontend/src/main/components/Leaderboard/LeaderboardTable.js

```diff
@@ -15,7 +20,10 @@ export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
         },
         {
             Header: 'Total Wealth',
-            accessor: 'totalWealth',
+            id: 'totalWealth',
+            accessor: (row, _rowIndex) => {
+                return USD.format(row.totalWealth);
+            }
```

Comment:
```suggestion
                return USD.format(row.totalWealth);
            }
            Cell: (props) => {
                return (
                  <div style={{textAlign: "right"}}>{props.value}</div>
                );
            },
        },
```

I think if you add this, you'll get right alignment.   

(based on digging through this [Stack overflow](https://stackoverflow.com/a/53605924) answer and some experimentation).

Reply:
Thank you! I completely missed this criteria. Updated.

<img width="922" alt="Screenshot 2023-08-11 at 4 16 09 PM" src="https://github.com/ucsb-cs156-m23/proj-happycows-m23-10am-4/assets/41165204/6d85dacf-c5bc-47fd-afef-1e55338fca0d">